radio/src/fonts.h

```diff
@@ -36,7 +36,7 @@ void loadFontCache();
 extern const pm_uchar font_5x7[];
 extern const pm_uchar font_10x14[];
 
-#if defined(BOLD_FONT) && (!defined(CPUM64) || defined(TELEMETRY_NONE)) && !defined(BOOT)

```

Comment:
I made it because, actualy, if we need to modify the condition for defining BOLD_SPECIFIC_FONT, we don't have to forget to do the same in lcd.cpp.
With this mod, there is only one modification to do, at the "root" of the font definition, and the consequences follow naturaly.